Modules/Scripted/Home/HomeLib/dependency_installer.py

```diff
@@ -59,4 +65,9 @@ def monai_pre_install():
 check_and_install_itk = functools.partial(check_and_install_package, ["itk"], "itk")
 check_and_install_pandas = functools.partial(check_and_install_package, ["pandas"], "pandas")
 check_and_install_matplotlib = functools.partial(check_and_install_package, ["matplotlib"], "matplotlib")
-check_and_install_monai = functools.partial(check_and_install_package, ["monai", "skimage", "tqdm"], "monai[skimage,tqdm]", monai_pre_install)
+check_and_install_monai = functools.partial(
+    check_and_install_package,
+    ["monai", "skimage", "tqdm", "PIL", "ignite", "gdown", "nibabel", "monai.transforms", "monai.deploy.core"],
```

Comment:
The example I started with, https://docs.monai.io/projects/monai-deploy-app-sdk/en/latest/notebooks/tutorials/02_mednist_app.html, requires `ignite` and `gdown`.  It also requires `nibabel` even though they don't mention it explicitly -- but see https://github.com/Project-MONAI/monai-deploy-app-sdk/issues/302.

With `ignite`, I do see the explicit use  in `02_mednist_app.html`, so it is possible that we don't need it because we don't use it explicitly.  With `gdown` and `nibabel`, I don't see the explicit use, seemingly telling us that it is within the `monai-deploy` implementation that these are needed and that we will need them in our case too.  But that's easy to check; I can experiment removing these dependencies and see what happens.

Reply:
Thanks for checking!

Reply:
We don't need `ignite`, `gdown`, and `nibabel`.  The commit that I just pushed now omits them.